roles/gandi.rb

```diff
@@ -0,0 +1,24 @@
+name "gandi"
+description "Role applied to all servers at Gandi"
+
+default_attributes(
+  :hosted_by => "Gandi",
+  :location => "Bissen, Luxembourg",
+  :networking => {
+    :roles => {
+      :external => {
+        :zone => "gandi"
```

Comment:
Just use `osm` as the zone - some of the older nodes do have a provider specific zone but I'm no longer doing that other than for the big sites with lots of machines and if you do add a new zone like that then the networking cookbook needs changing to define it.